site/src/docs/components/header-universal-bar/tabs.mdx

```diff
@@ -0,0 +1,35 @@
+---
+slug: '/components/header-universal-bar/tabs'
+title: 'HeaderUniversalBar'
+---
+
+import { StatusLabel, Notification } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# HeaderUniversalBar
+
+<StatusLabel type="alert">WIP</StatusLabel>
+<StatusLabel type="success" style={{ marginLeft: 'var(--spacing-xs)' }}>
+  Accessible
+</StatusLabel>
+
+<LeadParagraph>A header universal bar provides useful quick links for the user of the service.</LeadParagraph>
+
+<Notification label="A work in progress!" className="siteNotification">
+  The header universal bar component is part of a collection of new navigation components that the HDS team is in the
```

Comment:
Grammarly suggestion: "The navigation link component is part of a collection of new navigation components the HDS team is currently making. This means that this component is subject to change, and we don't recommend using it in production."